samples/dashboards/demo/datagrid-sync/demo.js

```diff
@@ -1,18 +1,52 @@
 const csvData = document.getElementById('csv').innerText;
 
+let chartCount = 0;
+
 const chartOptions = {
     xAxis: {
         type: 'category'
     },
+    yAxis: {
+        title: {
+            text: null
+        }
+    },
     chart: {
         animation: false,
-        type: 'column'
+        type: 'column',
+        events: {
+            load: function () {
+                const chart = this;
+                if (chartCount === 1) {
```

Comment:
You don't need to use counter. You can remove the whole load event and use spread operator in component chartOptions.

See the comment below.